Gateway/Http/Client/TransactionRefund.php

```diff
@@ -64,6 +64,12 @@ public function __construct(
         $client->setUsername($webserviceUsername);
         $client->setPassword($webservicePassword);
 
+        $client->setAdyenPaymentSource($this->_adyenHelper->getModuleName(), $this->_adyenHelper->getModuleVersion());
+
+		$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+		$productMetadata = $objectManager->get('Magento\Framework\App\ProductMetadataInterface');
+		$client->setExternalPlatform($productMetadata->getName(), $productMetadata->getVersion());
```

Comment:
Let's not use objectManager, best to use dependency injection in the constructor.